partcad/src/partcad/shape.py

```
@@ -450,6 +450,38 @@ def render_step(
+                from OCP.BRepTools import BRepTools
+
+                if not project is None:
+                    project.ctx.ensure_dirs_for_file(filepath)
+
+                brep_writer = BRepTools()
+                with open(filepath, "wb") as brep_file:
+                    brep_writer.Write_s(obj, brep_file)
+
+            await pc_thread.run(do_render_brep)
+
+    def render_brep(
```

Ruff (0.8.2)

Missing return type annotation for public function `render_brep`

Add return type annotation: `None`

(ANN201)